ccmlib/scylla_docker_cluster.py

```diff
@@ -48,6 +49,8 @@ def _update_config(self):
 class ScyllaDockerNode(ScyllaNode):
     def __init__(self, *args, **kwargs):
         kwargs['save'] = False
+        # This line should appear before the "super" method
+        self.share_directories = ['data', 'commitlogs', 'conf', 'hints', 'view_hints', 'saved_caches', 'keys', 'logs']
```

Comment:
please remove `conf` from here

Reply:
why? we want it mapped as a volume into the node in the end, no?

Reply:
we don't want the directory mapped, only specific files

Reply:
I cannt.
I got error when I did it

Reply:
you still need to add here something like `conf/<conf_file>`
otherwise, it won't work